docs/products/clickhouse/howto/connect-with-go.rst

```diff
@@ -0,0 +1,184 @@
+        },
+    })
+    if err != nil {
+        return err
+    }
+    v, err := conn.ServerVersion()
+    fmt.Println(v)
+
+Connect with HTTP
+-----------------
+
+.. important::
+
+    The HTTP connection is supported for the database/sql API only. By default, connections are established over the native protocol. The HTTP connection needs to be enabled either by modifying the DSN to include the HTTP protocol or by specifying the protocol in the connection options.
+
+Identify connection information
+'''''''''''''''''''''''''''''''
+
+To run the code for connecting to your service, first identify values of the following variables:
+
+==================      =====================================================================
+Variable                Description
+==================      =====================================================================
+``Host``                **Host** for the ClickHouse connection available in the Aiven console: Service **Overview** > **Connection information** > **ClickHouse HTTPS & JDBC**
+``HttpPort``                **Port** for the ClickHouse connection available in the Aiven console: Service **Overview** > **Connection information** > **ClickHouse HTTPS & JDBC**
+``Database``            **Database Name** in your the ClickHouse service available in the Aiven console: Service **Overview** > **Connection information** > **ClickHouse HTTPS & JDBC**
+``Username``            **User** for the ClickHouse connection available in the Aiven console: Service **Overview** > **Connection information** > **ClickHouse HTTPS & JDBC**
+``Password``            **Password** for the ClickHouse connection available in the Aiven console: Service **Overview** > **Connection information** > **ClickHouse HTTPS & JDBC**
+==================      =====================================================================
+
+Connect to the service
+''''''''''''''''''''''
+
+Replace the placeholders in the code with meaningful information on your service connection and run the code.
+
+.. code:: go
+
+    func ConnectHTTP() error {
+        env, err := GetStdTestEnvironment()
+        if err != nil {
+            return err
+        }
+        conn := clickhouse.OpenDB(&clickhouse.Options{
+            Addr: []string{fmt.Sprintf("%s:%d", env.Host, env.HttpPort)},
+            Auth: clickhouse.Auth{
+                Database: env.Database,
+                Username: env.Username,
+                Password: env.Password,
+            },
+            Protocol: clickhouse.HTTP,
+        })
+        return conn.Ping()
+    }
+
+    func ConnectDSNHTTP() error {
+        env, err := GetStdTestEnvironment()
+        if err != nil {
+            return err
+        }
+        conn, err := sql.Open("clickhouse", fmt.Sprintf("http://%s:%d?username=%s&password=%s", env.Host, env.HttpPort, env.Username, env.Password))
+        if err != nil {
+            return err
+        }
+        return conn.Ping()
+    }
+
+Established a configured connection
```

Comment:
I would remove the entire section about configured connection and redirect them to the original doc, it's very advanced and not really needed to succeed in using ClickHouse.